### DISCUSSION TITLE
Top-langs to include Repo I don't own

### DISCUSSION THREAD

**Discussion by EliseCho13** [anuraghazra/github-readme-stats#3547] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

I usually work as a team, rather than my personal repos.
However, the top-langs are only counting my own repos.
Can I include a repo I don't own but participated in as a team?
The chart looks so wrong as it only includes the repos I own.

### Describe the solution you'd like

Is there any other way to include the contributions I made to other repos in top-langs?
If there's no query for that, could you make it?

**Comment by qwerty541** [comment, marked as answer]
Hey, @EliseCho13! Top languages card shows language usage only inside your own non-forked repositories, not depending on who the author of the commits is. It does not include your contributions into another users/organizations repositories. Currently there are no way to get this data from GitHub API. If you want this behavior to be improved you can support [this feature request](https://github.com/orgs/community/discussions/18230) created by [@rickstaa](https://github.com/rickstaa) inside GitHub Community.
